src/vivarium/framework/values.py

```diff
@@ -203,24 +202,18 @@ class Pipeline:
     values that won't be used in the particular simulation.
     """
 
-    def __init__(self) -> None:
-        """
-        Parameters
-        ----------
-        name
-            The name of the value represented by this pipeline.
-        mutators
-            A list of callables that directly modify the pipeline source or
-            contribute portions of the value.
-        post_processor
-            An optional final transformation to perform on the combined output of
-            the source and mutators.
-        """
-        self.name: str | None = None
+    def __init__(self, name: str) -> None:
+        self.name: str = name
+        """The name of the value represented by this pipeline."""
```

Comment:
Maybe I'm forgetting, but isn't the original docstring correct since __init__ is a proper function and we only do this type of docstring for special-case things like properties?

Reply:
TBH I don't remember what we decided on officially. I think this is more maintainable, since it puts the docstring next to the entity it is documenting, but happy to revert if we prefer the other way.